api/api.go

```diff
@@ -545,6 +543,8 @@ func (a *ApplicationHandler) BuildDataPlaneRoutes() *chi.Mux {
 		uiRouter.Use(middleware.JsonResponse)
 		uiRouter.Use(chiMiddleware.Maybe(middleware.RequireAuth(), shouldAuthRoute))
 
+		uiRouter.Get("/license_features", handler.GetLicenseFeatures)
```

Comment:
this is a nitpick, but shouldn't this be
```suggestion
		uiRouter.Get("/licenses/{id}/features", handler.GetLicenseFeatures)
		
		or 
		
		uiRouter.Get("/licenses/features", handler.GetLicenseFeatures)
```